Update Node version in Travis config

Сейчас в тревисе задана версия 14, а в локальном `.nvmrc` указана 18. Видимо, из-за этого при попытке билда повылезали  ошибки вида:

```
$ node --version
v14.21.3
$ npm --version
6.14.18
$ nvm --version
0.39.4
4.23s$ npm ci 
npm ERR! Cannot read property 'chart.js' of undefined
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2023-11-12T17_58_58_311Z-debug.log
The command "eval npm ci  " failed. Retrying, 2 of 3.
npm ERR! Cannot read property 'chart.js' of undefined
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2023-11-12T17_58_59_725Z-debug.log
The command "eval npm ci  " failed. Retrying, 3 of 3.
npm ERR! Cannot read property 'chart.js' of undefined
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2023-11-12T17_59_01_131Z-debug.log
The command "eval npm ci  " failed 3 times.
The command "npm ci " failed and exited with 1 during .
Your build has been stopped.
```

Пробую перейти везде на 18 версию.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - 14
+  - 18
 
 branches:
   only:

